code/stable/gooltest/java/HelloWorld/HelloWorld/HelloWorld.java

```diff
@@ -6,9 +6,9 @@
     \note Generated by Drasil v0.1-alpha
 */
 
+import java.util.*;
```

Comment:
Is this really the right thing to import? Doesn't this make the imports below this redundant?

Reply:
This is from old changes I made and is redundant now, I will remove it.